pkg/controller/gittrackobject/handler.go

```diff
@@ -234,6 +234,7 @@ func (r *ReconcileGitTrackObject) applyChildWithDryRun(found, child *unstructure
 	dryRunTrue := true
 	err := r.applier.Apply(context.TODO(), &farosclient.ApplyOptions{ForceDeletion: &force, ServerDryRun: &dryRunTrue}, child)
 	if err != nil {
+		fmt.Printf("DryRun failed: %+v\n", err)
```

Comment:
Just noticed this extra logging, as of #108 (which I just merged), the error returned below would be propagated up and printed out, so I think this is no longer needed. Would you be able to remove this line and the print below as well?

Reply:
Done, thanks for noticing and the improvement in #108